External/pubchem_shape/PubChemShape.hpp

```diff
@@ -2,11 +2,47 @@
 #include <map>
 #include <vector>
 
+#include <boost/archive/text_oarchive.hpp>
```

Comment:
a couple of notes here:
- boost::serialization support is optional, so all the added code needs to be inside a check for `RDK_USE_BOOST_SERIALIZATION`
- these includes should be inside the `#include <RDGeneral/BoostStartInclude.h>` block below